recipes/autoconf/all/conanfile.py

```diff
@@ -1,4 +1,5 @@
 from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conan.tools.microsoft import unix_path
```

Comment:
You need to bump the `required_conan_version` below.
On the other hand, older conan versions will fail at this line.